### ISSUE TITLE
Version confusion

### ISSUE THREAD

**Issue by FreakTheMighty** [jannessm/quadric-mesh-simplification#8]
Hi @jannessm 👋 . I'm a big confused about the 1.1.1 version published to pypi. 

If I try to install 1.1.1, I get the old version

```bash
$ pip install quad_mesh_simplify==1.1.1
```
> Collecting quad_mesh_simplify==1.1.1
>   Using cached quad_mesh_simplify-1.1.1-cp36-cp36m-manylinux1_x86_64.whl (105 kB)
> Requirement already satisfied: numpy in /usr/local/lib/python3.6/site-packages (from quad_mesh_simplify==1.1.1) (1.18.1)
> Requirement already satisfied: cython in /usr/local/lib/python3.6/site-packages (from quad_mesh_simplify==1.1.1) (0.29.21)
> Installing collected packages: quad-mesh-simplify
> Successfully installed quad-mesh-simplify-1.1.0

It took some digging through pypi, but then I noticed some of the pypi artifacts seem to be published to `1.1.1a`.

```bash
$ pip install quad_mesh_simplify==1.1.1a
```

> Collecting quad_mesh_simplify==1.1.1a
>   Using cached quad_mesh_simplify-1.1.1a-cp36-cp36m-manylinux1_x86_64.whl (105 kB)
> Requirement already satisfied: cython in /usr/local/lib/python3.6/site-packages (from quad_mesh_simplify==1.1.1a) (0.29.21)
> Requirement already satisfied: numpy in /usr/local/lib/python3.6/site-packages (from quad_mesh_simplify==1.1.1a) (1.18.1)
> Installing collected packages: quad-mesh-simplify
> Successfully installed quad-mesh-simplify-1.1.1

----

So I guess the question is, did something go wrong on publishing? Should the package version be changed?

**Comment by FreakTheMighty** [comment]
It seems to come down to a mismatch between setup.py and METADATA https://github.com/pypa/pip/issues/8917#issuecomment-699442712

**Comment by jannessm** [comment]
Sorry, that it took me so long to answer. I will publish a new version in the next days. So please be patient.

**Comment by jannessm** [comment]
Sorry that it took so long! I have now published version 1.1.2 to remove the version confusion. You should be able now to install it without any problems.
